core/src/wallet/algorand/AlgorandAccount.cpp

```diff
@@ -94,26 +96,36 @@ namespace algorand {
 
     bool Account::putBlock(soci::session& sql, const api::Block& block)
     {
-        if (BlockDatabaseHelper::putBlock(sql, block)) {
-            emitNewBlockEvent(block);
-            return true;
-        }
-        return false;
+        emitNewBlockEvent(block);
+        return true;
     }
 
-    int Account::putTransaction(soci::session &sql, const model::Transaction &transaction)
+    void Account::interpretTransaction(const model::Transaction &transaction, std::vector<Operation> &out)
     {
+        soci::session sql;
         const auto wallet = getWallet();
         if (wallet == nullptr) {
             throw Exception(api::ErrorCode::RUNTIME_ERROR, "Wallet reference is dead.");
         }
-        const auto txuid = TransactionDatabaseHelper::putTransaction(sql, getAccountUid(), transaction);
+
         const auto operation = Operation(shared_from_this(), transaction);
+        out.push_back(operation);
 
-        if (OperationDatabaseHelper::putAlgorandOperation(sql, txuid, operation)) {
```

Comment:
Since `algorand::OperationDatabaseHelper::putAlgorandOperation` isn't used anymore you can probably remove `AlgorandOperationDatabaseHelper.hpp` and `AlgorandOperationDatabaseHelper.cpp` entirely?

Reply:
done